api/src/paths/codes.ts

```diff
@@ -349,6 +350,25 @@ GET.apiDoc = {
                     }
                   }
                 }
+              },
+              attractants: {
```

Comment:
Title / description specs would be helpful here

Reply:
Added some stuff, though I probably need to go through and do the same for all of the codes, maybe as tech debt separately.